pkg/secrets/repositoryserver/aws_secrets_test.go

```diff
@@ -16,11 +16,13 @@ package repositoryserver
 
 import (
 	"github.com/pkg/errors"
-	. "gopkg.in/check.v1"
+
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	secerrors "github.com/kanisterio/kanister/pkg/secrets/errors"
+
+	. "gopkg.in/check.v1"
```

Comment:
This one should be reverted. The import order was correct before the change.

```suggestion
	. "gopkg.in/check.v1"
	corev1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"

	secerrors "github.com/kanisterio/kanister/pkg/secrets/errors"
```